components/server/src/express/ws-handler.ts

```diff
@@ -64,13 +64,13 @@ export class WsExpressHandler {
         const stack = WsLayer.createStack(...handlers);
         const dispatch = (ws: websocket, request: express.Request) => {
             handler(ws, request);
-            stack.dispatch(ws, request).catch(err => {
-                log.error("websocket stack error", err);
-                ws.terminate();
-            }).finally(() => {
+            stack.dispatch(ws, request).finally(() => {
                 const pathname = request.url ? url.parse(request.url).pathname : undefined;
                 const method = request.method || "UNKNOWN";
                 increaseHttpRequestCounter(method, pathname || "unkown-websocket", request.statusCode || 0);
+            }).catch(err => {
+                log.error("websocket stack error", err);
+                ws.terminate();
```

Comment:
Out of curiosity, why put the `.catch()` after the `.finally()` and not the other way around?

Reply:
`finally` can throw (again), that's why the lint wants the `catch` to be the last combinator.